server/routes/posts.js

```diff
@@ -151,11 +152,12 @@ router.post("/:post_id/likes", function(req, res) {
 });
 
```

Comment:
take the latest. delete route missing

Reply:
What does this mean?

Reply:
there should be a route to "delete post". which is missing.

Reply:
@Badr-KH make sure all our changes are there before the demo

Reply:
This was put up 5 days ago.  Your route was not yet written.  The merge should be unaffected by your addition.